### PR TITLE
Remove Params in Scheme files

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: json/src/scheme/params.rs [hunk 1]
```diff
@@ -1,80 +0,0 @@
-// Copyright 2018-2020 Kodebox, Inc.
```
Comment (majecty) [inline json/src/scheme/params.rs:1]: I thought about whether remove this or not.

We may need this code to serialize Consensus?  params.

We don't need it now. This file is in scheme folder, We may rewrite similar code easily.
So I removed this.

**Comment by junha1** [comment]
There is a fmt error

**Comment by majecty** [comment]
@junha1 Thanks, I fixed them.

File: json/src/scheme/params.rs (hunk 1 above)
Comment (junha1) [reply json/src/scheme/params.rs:1]: ok.

**Review by junha1 — APPROVED** [review]

**Comment by majecty** [comment]
I'll merge it. `cargo run test` succeeds in my local machine.

**Pull request merged**